includes/admin/functions.php

```diff
@@ -368,7 +368,7 @@ function add_customizer_messaging_panel( $wp_customize ) : void {
 		$partial_message_id,
 		[
 			'capability'        => apply_filters( 'coil_settings_capability', 'manage_options' ),
-			'sanitize_callback' => 'wp_filter_nohtml_kses',
+			'sanitize_callback' => 'esc_textarea',
```

Comment:
esc_textarea is an escaping function, not a sanitisation function. This won't be safe to use here.

You should do some research and test this but I think `sanitize_text_field` might be appropriate here.